localstack/services/sns/provider.py

```diff
@@ -352,8 +352,18 @@ def create_platform_endpoint(
         return result
 
     def unsubscribe(self, context: RequestContext, subscription_arn: subscriptionARN) -> None:
-        call_moto(context)
-        store = self.get_store(account_id=context.account_id, region_name=context.region)
+        try:
+            parsed_arn = parse_arn(subscription_arn)
+        except InvalidArnException:
+            raise InvalidParameterException("Invalid parameter: SubscriptionId")
+
+        account_id = parsed_arn["account"]
+        region_name = parsed_arn["region"]
+
+        moto_sns_backend = self.get_moto_backend(account_id, region_name)
+        moto_sns_backend.unsubscribe(subscription_arn)
```

Comment:
It's nice that moto does not raise any kind of exception here, so it's pretty safe to use! Perfect 😄